src/colmap/scene/image_test.cc

```diff
@@ -35,7 +35,7 @@ namespace colmap {
 namespace {
 
```

Comment:
It will be great to have some minimal tests for the new Image class.

Reply:
Added here: https://github.com/colmap/colmap/pull/2702/commits/9a4c478a51a1e772fe3ac511b8ea9e69a955e3f7